### ISSUE TITLE
test_asyncio failed: flaky `test_sendfile_close_peer_in_the_middle_of_receiving`

### ISSUE THREAD

**Issue by tasty0tomato** [python/cpython#110325]
I built 3.11.5 on my own Windows computer, and found it passed this test on one computer, but didn't pass on another computer. I think that this issue may not be fixed completely, or I can provide needed information of the computer which this test didn't pass to find out the reason why it didn't pass.

(Originally posted in https://github.com/python/cpython/issues/85848#issuecomment-1744118631)

**Comment by sobolevn** [comment]
Please, provide more information: os name and version, test output, how python was built and executed.

**Comment by tasty0tomato** [comment]
OS name: Windows 10

OS version: 19045.3516

Test output(only _test_asyncio_):
```
0:00:44 [ 26/434] test_asyncio
test test_asyncio failed -- Traceback (most recent call last):
  File "C:\Python-3.11.5\Lib\test\test_asyncio\test_sendfile.py", line 473, in test_sendfile_close_peer_in_the_middle_of_receiving
    self.assertTrue(1024 <= self.file.tell() < len(self.DATA),
AssertionError: False is not true : 0
```

How python was built:
Just followed _PCbuild/readme.txt_, ran _build.bat_.

**Comment by sobolevn** [comment]
It has a big history: https://github.com/search?q=repo%3Apython%2Fcpython+test_sendfile_close_peer_in_the_middle_of_receiving&type=issues

Thanks for the details, this test is known to be problematic. You can just ignore it for now.
It should be fixed on our side :)

**Comment by gvanrossum** [comment]
Wasn't this fixed?

**Comment by tasty0tomato** [comment]
Yes, at least this problem is fixed in 3.11.7, but I find another issue in 3.11.7 which has been reported in https://github.com/python/cpython/issues/113026.

**Comment by tasty0tomato** [comment]
Well, because of some reasons, I need to build a python3.8.19. And it also has this kind of errors. But it seems that python 3.8 will be eol by the end of this year. So, anyway.
